main.py

```diff
@@ -132,6 +141,21 @@
 
 ############################################################################################
 
+# Open the config file and parse game configuration
+# This may overwrite configuration above
+try:
+    config_data     = config.open_config()
+    # Gamestate
+    Gamestate       = config_data['Gamestate']
```

Comment:
All of this would only work if all of these values are present. I'm there config. Wouldn't it make more sense to gracefully handle keys being missing?

Reply:
Agreed. I'd have to think a little on how to set that up without doing try/except everywhere. We want less clutter, not more.

Reply:
My money says `config_data.get('Gamestate', 'none')` will work. Assuming the yaml lib is doing what I think is doing under the hood

Reply:
This does work. Will make the changes.

Reply:
You should be able to do something like:

```python
try:
    config_data = config.open_config()
except WhateverSpecificExceptionHappensWhenConfigFailsToLoad:
    config_data = dict()
    print('Config data failed to load')
Gamestate = config_data.get('Gamestate', 'none')
...
```

This ought to deal with all the potential issue of the config not being there, values not being in there, etc.